scripts/evaluator_execution.py

```
@@ -0,0 +1,208 @@
+                    model=model,
+                    messages=cast(Any, list(task.messages)),
+                    **kwargs,
+                )
+                stats.record(getattr(completion, "usage", None))
+                raw_content = completion.choices[0].message.content
+                if raw_content is not None:
+                    content = str(raw_content).strip()
+                    break
+                print(f"  Empty content (attempt {attempt + 1})")
+                sleep_fn(retry_backoff_base**attempt)
+            except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)